modules/porous_flow/src/materials/PorousFlowPermeabilityConst.C

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowPermeabilityConst.h"
+
+template<>
+InputParameters validParams<PorousFlowPermeabilityConst>()
+{
+  InputParameters params = validParams<Material>();
+
```

Comment:
Sorry, that comment went crazy.  The line should be:

InputParameters params = validParams<PorousFlowPermeaiblityUnity>()

Ie, get the params from the "Unity" version, and then add your extra ones on top of them